package.json

```diff
@@ -132,7 +133,6 @@
 		"mapbox-gl": "0.54.1",
 		"markdown-it": "8.4.2",
 		"node-sass": "4.12.0",
-		"object-hash": "2.0.0",
```

Comment:
Is `object-hash` used elsewhere in Jetpack or did we introduce it?

Reply:
We (I) introduced it.